provisions/roles/scanner/tasks/main.yml

```diff
@@ -27,6 +27,10 @@
     regexp="^#*\s*ADD_REGISTRY=.*"
     replace='ADD_REGISTRY="--add-registry {{ public_registry }} --insecure-registry {{ public_registry }}:5000"'
 
+- name: Set SELinux to permissive
+  selinux:
+      state: permissive
+
```

Comment:
@navidshaikh because we're mounting Docker socket `/var/run/docker.sock` inside the container that's going to scan the image. Using which a new container is spunn off to check for respective package manager updates.